### PR TITLE
Bug/restore video service

### PR DESCRIPTION
**Pull request by jakeknott**

### CODE REVIEW

File: client/app/tab/tab-content.component.html [hunk 1]
```diff
@@ -12,127 +12,117 @@
...
+							submitButtonLabel="Delete"
+							modalClass="btn btn-info btn-lg"
+							[hideCloseButton]="true"
+							[closeOnEscape]="true"
+							[closeOnOutsideClick]="true"
+							(onSubmit)="DeleteModal.close()"
+							(onSubmit)="deleteVideoAnswer()">
+					
+							<modal-header>
+								This will perminatly delete your saved video!
```
Comment (Kcaruso) [inline client/app/tab/tab-content.component.html]: typo in "permanently"

File: client/app/tab/tab-content.component.html [hunk 2]
```diff
@@ -12,127 +12,117 @@
...
+							</modal-header>
+						
+							<modal-content>
+								By clicking 'Delete', you will perminatly remove your saved video, this action cannot
```
Comment (Kcaruso) [inline client/app/tab/tab-content.component.html]: typo in "permanently"

File: client/app/tab/tab-content.component.ts [hunk 3]
```diff
@@ -291,19 +215,47 @@ export class TabContent implements OnInit {
...
+		{
+			return;
+		}
+
+		this._videoService.deleteAnswer(this.answerVideo._id)
+		.then( result => {
+			this.setQuestionAndAnswer(this.selectedQuestion[0]._id);
+		});
+
+		var avid = videojs("avideo");
```
Comment (Kcaruso) [inline client/app/tab/tab-content.component.ts]: im being picky, but can you change these to "let"? having multiple js variable with the same name isn't great if we can avoid it. If your tool support es6 that is. If not, don't worry about it

**Comment by jakeknott** [comment]
Made changes.

**Review by Kcaruso — APPROVED** [review]
Looks good to merge

**Pull request merged**